stripe/api_resources/customer.py

```diff
@@ -20,63 +20,3 @@
 class Customer(CreateableAPIResource, UpdateableAPIResource,
                ListableAPIResource, DeletableAPIResource):
     OBJECT_NAME = 'customer'
-
```

Comment:
@ob-stripe do we want to remove the `stripe.Customer.*_source()` methods?

Reply:
No, those were recently added and are useful (they let users manage payment sources without retrieving the customer object first). We should try to document them somewhere though.

Reply:
I know they were added but I do wonder if there should be a different way to handle them. Deferring to you here obviously as I know you added them and know the library better than I do.